spec/03-credentials/08-SharedFileCredentials_spec.lua

```diff
@@ -24,12 +25,15 @@ describe("SharedFileCredentials_spec", function()
   local SharedFileCredentials_spec
 
   before_each(function()
+    -- make ci happy
+    restore.setenv("HOME", "/home/ci-test")
```

Comment:
since this is a library, it should also work on Windows, so in the final solution we should also cater for the Windows env vars.
(here is doesn't matter, since its only for testing)